docs/source/user-guide/tutorials/build-pkgs-skeleton.rst

```diff
@@ -0,0 +1,376 @@
+populated with information from the PyPI metadata and in many cases will not
+need to be edited.
+
+Files in the folder with meta.yaml are collectively referred to as the "conda
+build recipe":
+
+* ``meta.yaml``---Contains all the metadata in the recipe. Only
+  the package name and package version sections are
+  required---everything else is optional.
+
+* ``bld.bat``---Windows commands to build the package.
+
+* ``build.sh``---macOS and Linux commands to build the package.
+
+Now that you have the conda build recipe ready, you can use conda
+build to create the package:
+
+.. code-block:: bash
+
+    conda-build click
+
+When conda build is finished, it displays the exact path and
+filename of the conda package. See :ref:`troubleshooting` if the
+``conda-build`` command fails.
+
+Windows example file path:
+
+.. code-block:: text
+
+    C:\Users\jsmith\miniconda\conda-bld\win-64\click-7.0-py37_0.tar.bz2
+
+OS X example file path:
+
+.. code-block:: text
+
+    /Users/jsmith/miniconda/conda-bld/osx-64/click-7.0-py37_0.tar.bz2
+
+
+Linux example file path:
+
+.. code-block:: text
+
+    /home/jsmith/miniconda/conda-bld/linux-64/click-7.0-py37_0.tar.bz2
+
+
+.. note::
+   Your path and filename will vary depending on your
+   installation and operating system. Save the path and filename
+   information for the next step.
+
+Now you can install your newly-built package in your conda
+environment by using the use-local flag:
+
+.. code-block:: bash
+
+    conda install --use-local click
+
+Notice that Click is coming from the local conda build channel.
+
+.. code-block:: bash
+   
+   (click) 0561:~ jsmith$ conda list
+   # packages in environment at /Users/Jsmith/miniconda/envs/click:
+   # Name                    Version                   Build  Channel
+   ca-certificates           2019.1.23                     0 
+   certifi                   2019.3.9                 py37_0 
+   click                     7.0                      py37_0    local
+
+Now verify that Click installed successfully:
+
+.. code-block:: bash
+
+    conda list
+
+Scroll through the list until you find Click.
+
+At this point you now have a conda package for Click that
+can be installed in any conda environment of the same Python
+version as your root environment. The remaining optional sections
+show you how to make packages for other Python versions and other
+architectures and how to upload them to your Anaconda.org account.
+
+
+.. _`python-versions`:
+
+Optional---Building for a different Python version
+==================================================
+By default, conda build creates packages for the version of
+Python installed in the root environment. To build packages for
+other versions of Python, you use the ``--python`` flag followed
+by a version. For example, to explicitly build a version of the
+Click package for Python 3, use:
+
+.. code-block:: bash
+
+    conda-build --python 3 click
```

Comment:
I think the reason why @soapy1 recommended python 2.7 is that it is different from what we show above.  You can supply any python version, even inexact ones like 2 or 3.  Unfortunately, this example doesn't show any capability here, because the outputs are the same as the packages above (all python 3.7)

Reply:
I changed it back to 2.7.